chsdi/static/doc/source/services/oerebservices.rst

```diff
@@ -16,9 +16,13 @@ The returned format is Interlis (XML).
 .. warning::
   This service is only available for the following layers:
 
+  - ch.astra.projektierungszonen-nationalstrassen.oereb *(this layer has no data)*
+  - ch.bav.baulinien-eisenbahnanlagen.oereb *(this layer has no data)*
   - ch.bav.kataster-belasteter-standorte-oev.oereb
+  - ch.bav.projektierungszonen-eisenbahnanlagen.oereb *(this layer has no data)*
+  - ch.bazl.baulinien-flughafenanlagen.oereb
   - ch.bazl.kataster-belasteter-standorte-zivilflugplaetze.oereb
-  - ch.bazl.projektierungszonen-flughafenanlagen.oereb
+  - ch.bazl.projektierungszonen-flughafenanlagen.oereb *(this layer has no data)*
```

Comment:
Wrong layer! This one has data.
The layer without any objects in it is ch.bazl.baulinien-flughafenanlagen.oereb